src/components/CommentItem.css

```diff
@@ -0,0 +1,9 @@
+#user {
```

Comment:
These should be CSS classes, since there will be multiple of these on the page.

Reply:
I've put all the common CSS class properties in index.css instead and deleted the specific .css files that were redundant

Reply:
index.css was the first common root of the different containers using the same CSS classes